Client/Frontend/Settings/FxAContentViewController.swift

```diff
@@ -131,6 +131,7 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
         if profile.hasAccount() {
             LeanplumIntegration.sharedInstance.setUserAttributes(attributes: [UserAttributeKeyName.signedInSync.rawValue:  true])
         }
+        LeanplumIntegration.sharedInstance.track(eventName: LeanplumEventName.signsInFxa)
```

Comment:
I know this is a small change, but if that belongs to a different bug, please do two separate pull requests. We usually don't combine things like this, and also the commit message is way too long if yuo just append `& Bug ...` to it.